UniConnectApp/app/src/main/java/com/example/uniconnect/RetrofitClientInstance.kt

```diff
@@ -7,7 +7,7 @@ object RetrofitClientInstance {
     private var retrofit: Retrofit? = null
     private val BASE_URL = "http://universities.hipolabs.com/"
 
-    val retrofitInstance : Retrofit?
+    val retrofitInstance: Retrofit?
```

Comment:
automatic via kotlin style guide per https://kotlinlang.org/docs/coding-conventions.html#apply-the-style-guide